.storybook/stories/DataFiltering/TableFilteringMedium.tsx

```diff
@@ -22,23 +36,104 @@ import {
 } from './components/FilterBar';
 import { DashboardPagination } from './components/DashboardPagination';
 import { DataTableSelectAllCheckbox } from './components/DataTableSelectAllCheckbox';
+import { assignees, BusinessForAudit } from './lib/generateBusinessData';
+import { useDataContext } from './lib/contexts';
 
 const headingId = 'table-heading';
 
 export const TableFilteringMedium = ({
+	hasActionColumn,
 	selectable,
 }: {
+	hasActionColumn?: boolean;
 	selectable?: boolean;
 }) => {
+	const { updateData } = useDataContext();
+
+	const sectionAlertRef = useRef(null);
 	const [isOpen, toggleIsOpen] = useToggleState(false, true);
 
+	const [currentItem, setCurrentItem] = useState<BusinessForAudit>();
+	const [showSuccessMessage, setShowSuccessMessage] = useState(false);
+
+	const [cancelModalOpen, setDeleteAuditModalOpen] = useState(false);
+
+	const [isDrawerOpen, openDrawer, closeDrawer] = useTernaryState(false);
+	const onSubmitForm: MouseEventHandler<HTMLButtonElement> = (event) => {
+		event.preventDefault();
+
+		if (!currentItem) return;
+		updateData?.({
+			...currentItem,
+			...(radioAssigneeValue && { assignee: radioAssigneeValue }),
+		});
+
+		setShowSuccessMessage(true);
+
+		closeDrawer();
+	};
+	const onOpenDrawer = (newCurrentItem?: BusinessForAudit) => {
+		openDrawer();
+		setShowSuccessMessage(false);
```

Comment:
Feels weird removing all of these success messages through, because the noticeable UI jump makes it look like something else happened when the drawers and modal open

Reply:
We can remove them on cancel, instead? The form submit will update the text.

Reply:
Yeah, probably better